pallets/allocations/src/lib.rs

```diff
@@ -201,14 +245,9 @@ impl<T: Config> Pallet<T> {
 	}
 }
 
-impl<T: Config> ChangeMembers<T::AccountId> for Pallet<T> {
-	fn change_members_sorted(_incoming: &[T::AccountId], _outgoing: &[T::AccountId], new: &[T::AccountId]) {
-		<Oracles<T>>::put(new);
-	}
-}
-
-impl<T: Config> InitializeMembers<T::AccountId> for Pallet<T> {
-	fn initialize_members(init: &[T::AccountId]) {
-		<Oracles<T>>::put(init);
+#[cfg(feature = "runtime-benchmarks")]
+impl<T: Config> Contains<T::AccountId> for Pallet<T> {
+	fn contains(t: &T::AccountId) -> bool {
+		Self::validator_set().binary_search(t).is_ok()
```

Comment:
If you take my simplification suggestion, you can remove this.

Reply:
Done